diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/PackageNaming.kt

```diff
@@ -215,31 +216,31 @@ class PackageNaming(private val configRules: List<RulesConfig>) : Rule("package-
      * FixMe: check if proper line/char numbers are added
      */
     private fun createAndInsertPackageName(parentNode: ASTNode, insertBeforeNode: ASTNode?, packageNameToInsert: List<String>) {
-        var compositeElementWithNameAndDot: CompositeElement? = null
-        var childDot: LeafPsiElement? = null
-        var childPackageNamePart: LeafPsiElement?
-
-        packageNameToInsert.forEach { name ->
+        packageNameToInsert.map { name ->
             // creating Composite object = ((IDENTIFIER) + (DOT))
-            compositeElementWithNameAndDot = CompositeElement(REFERENCE_EXPRESSION)
-            childDot = LeafPsiElement(DOT, PACKAGE_SEPARATOR)
-            childPackageNamePart = LeafPsiElement(IDENTIFIER, name)
+            val compositeElementWithNameAndDot = CompositeElement(REFERENCE_EXPRESSION)
```

Comment:
I have removed this code at all

Reply:
could you please revert? Don't waste time on fixing conflicts

Reply:
Yes, done